src/main/webapp/find-events.html

```diff
@@ -64,7 +64,35 @@ <h1 id="find-events-title" tabindex="0">find events</h1>
           </div>
         </div>
         <div id="all-posts">
-          <button id="create-post-button" tabindex="0" type="button">create a post</button>
+          <div id="all-posts-buttons">
+            <button id="create-post-button" type="button" tabindex="0">create a post</button>
+            <button id="toggle-filters-button" type="button" tabindex="0">filters</button>
+          </div>
+          <div id="filter-card">
+            <div class="filter">
+              <label for="num-people" class="filter-label long-text">number of people the event can feed:</label>
+              <label for="num-people" class="filter-label short-text">number of people:</label>
+              <span class="no-wrap filter-slider">
+                0 <input type="range" min="0" max="40" value="0" id="num-people" name="num-people" /> 40+
```

Comment:
Is there a reason to have the max as 40? I'm wondering if 50 might be a more round number

Reply:
Also Im wondering how it looks with the `happening right now` at the top so the two sliders are together?

Reply:
I picked it pretty arbitrarily, I'll change it to 50! :)

Reply:
And I can also change the other slider up!

Reply:
![image](https://user-images.githubusercontent.com/35010111/90810918-9642e080-e2d8-11ea-9229-c8eff71b1f3b.png)
This is what it looks like now :)